case-lib/pipeline.sh

```diff
@@ -4,7 +4,7 @@ func_pipeline_export()
 {
     # no parameter input the function
     if [ $# -lt 1 ]; then
-        dlogi "Topology file name is not specified, unable to run command: ${BASH_SOURCE[-1]}" && exit 1
+        dloge "Topology file name is not specified, unable to run command: ${BASH_SOURCE[-1]}" && exit 1
```

Comment:
For the topology verification, I think It should be error, and we should make test case failed, rather than NA, NA will make people confuse under this condition.

Reply:
for the all case, it don't need to topology verification, it is different scope